s2core/src/main/scala/com/daumkakao/s2graph/core/parsers/WhereParser.scala

```diff
@@ -104,28 +116,36 @@ case class WhereParser(label: Label) extends JavaTokenParsers with JSONParser {
 
   def paren: Parser[Clause] = "(" ~> clause <~ ")"
 
-  def clause: Parser[Clause] = (predicate | paren) *
-    ("and" ^^^ { (a: Clause, b: Clause) => And(a, b) } |
-      "or" ^^^ { (a: Clause, b: Clause) => Or(a, b) })
+  def clause: Parser[Clause] = (predicate | paren) * ("and" ^^^ { (a: Clause, b: Clause) => And(a, b) } | "or" ^^^ { (a: Clause, b: Clause) => Or(a, b) })
 
   /** TODO: exception on toInnerVal with wrong type */
+  def extract(lv: String, rv: String) = metaProps.get(lv) match {
```

Comment:
lv, rv is not clear what they meant. did you mean lv to be property key and rv to be property value?